src/cli/utils/chains.js

```diff
@@ -38,14 +37,9 @@ const createChainFromConf = (
 ) => {
   try {
     const chain = { ...chainConf };
-    const provider =
-      chainConf.host && chainConf.host.indexOf('http') === 0
-        ? new JsonRpcProvider(chainConf.host, {
-            ensAddress: chainConf.ensRegistry,
-            chainId: parseInt(chainConf.id, 10),
-            name: chainName,
-          })
-        : getReadOnlyProvider(chainConf.host, { providers: providerOptions });
+    const provider = getReadOnlyProvider(chainConf.host, {
```

Comment:
Both parts of the ternary opertator created a Provider without signer (read-only)
The ternary only existed to inject the custom network (bellecour) with ENS capabilities.
Since bellecour is now added globally via `Network.register()` there is no more need for this code.